draft-ietf-taps-interface.md

```diff
@@ -299,10 +299,6 @@ provides:
 
 # API Summary
 
-The Transport Services API is the basic common abstract application
-programming interface to the Transport Services Architecture defined in the TAPS
-Architecture {{I-D.ietf-taps-arch}}.
-
```

Comment:
Why remove this - isn't the API the inerface spec for the architecture?

Reply:
See #899

Reply:
OK, understood -see issue.